packages/cli/src/commands/env/pull.ts

```diff
@@ -56,8 +56,12 @@ export default async function pull(
     return 1;
   }
 
+  // handle relative or absolute filename
   const [filename = '.env'] = args;
-  const fullPath = join(process.cwd(), filename);
+  let fullPath = isAbsolute(filename)
+    ? filename
+    : join(process.cwd(), filename);
+
```

Comment:
We can use `path.resolve()` here since it already handles cwd when given a relative path:

```suggestion
  const fullPath = resolve(filename);
```